### PR TITLE
Add compatability for old PowerShell versions

### PR DESCRIPTION
**Pull request by ButterscotchV**
Turns out older PowerShell versions *suck,* so this should fix a couple of the compatability issues
Works on Windows 8, but untested as of yet with modern Windows/PowerShell, will do that in a bit

### CODE REVIEW

**Comment by ButterscotchV** [comment]
This does work on Windows 11/PowerShell 7, though now it saves an unformatted `openvrpaths.vrpath` file, which is fine but just different than it originally had done. This could be worked around by handling the error and trying to use this format instead if the default fails? I'd be interested to hear people's input on that

**Comment by ButterscotchV** [comment]
This installer seems to have an issue with uninstalling, this might be a main installer issue?

File: windows/web/steamvr.ps1 [hunk 1]
```diff
@@ -35,13 +35,13 @@ if ($ExternalDriverPaths.Length -eq 0) {
 } else {
     $OpenVrConfig.external_drivers = $ExternalDriverPaths
 }
-[System.IO.File]::WriteAllLines($OpenVrConfigPath, (ConvertTo-Json -InputObject $OpenVrConfig))
+[System.IO.File]::WriteAllLines($OpenVrConfigPath, (ConvertTo-Json -InputObject $OpenVrConfig -Compress))
```
Comment (deiteris) [inline windows/web/steamvr.ps1:38]: Why do we need `-Compress` again?

Comment (ButterscotchV) [reply windows/web/steamvr.ps1:38]: Some strange PowerShell bug, without that it throws a JSON format error
```
+ [System.IO.File]::WriteAllLines($OpenVrConfigPath, (ConvertTo-Json 
-InputObject  ...
+                                                     
~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (@{config=System...t[]; versio 
   n=1}:PSObject) [ConvertTo-Json], InvalidOperationException
    + FullyQualifiedErrorId : JsonStringInBadFormat,Microsoft.PowerShell.Comma 
   nds.ConvertToJsonCommand
```
If I remember correctly, I saw something mention it had to do with trailing backslashes in strings causing that, and the file usually has a path or two with trailing backslashes.

**Review by deiteris — APPROVED** [review]

**Pull request merged**